myRental/app.js

```diff
@@ -0,0 +1,37 @@
+var express = require('express');
```

Comment:
Setup Babel with the server so that we can use ES6/ES7 JavaScript (https://babeljs.io)

Reply:
I looked into this and I dont think we need to use babel. The reason you would use babel for react is that react sends javascript to the browser and the browser only supports a limited number of languages. However on the Express side the javascript is given to a Node process to execute and Node supports the latest version of javascript.

So let me know if you see something that can be done better with the new standard and I will change anything over.